internal/sorter/sorter_test.go

```diff
@@ -507,10 +619,7 @@ func TestSortClusters(t *testing.T) {
 		{Name: "second"},
 	}
 
-	have := []*envoy_cluster_v3.Cluster{
-		want[1],
-		want[0],
-	}
+	have := shuffleSlice(want)
```

Comment:
I like this change in general but with only 2 items in the slice, we have a 50-50 chance of the shuffled slice already being ordered correctly. Maybe we should run all of these tests that involve randomness a bunch of times to avoid any flakes due to that. Could just wrap the shuffles + asserts in a loop that does it 10 times.